### PR TITLE
Typos & Style clean up

### PR DESCRIPTION
**Pull request by Piinks**
## Description

Recently working in this area, noticed an opportunity to tidy. I separated the  clean-up from the actual change (#41327) for easier review.

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require Flutter developers to manually update their apps to accommodate your change?

- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

File: packages/flutter/lib/src/painting/text_span.dart [hunk 1]
```diff
@@ -246,7 +254,10 @@ class TextSpan extends InlineSpan {
     }
     if (children != null) {
       for (InlineSpan child in children) {
-        assert(child is TextSpan, 'visitTextSpan is deprecated. Use visitChildren to support InlineSpans');
+        assert(
+          child is TextSpan,
+        'visitTextSpan is deprecated. Use visitChildren to support InlineSpans',
```
Comment (chunhtai) [inline packages/flutter/lib/src/painting/text_span.dart]: the style is a bit off

File: packages/flutter/lib/src/painting/text_span.dart [hunk 2]
```diff
@@ -322,7 +337,8 @@ class TextSpan extends InlineSpan {
 
   @override
   void describeSemantics(Accumulator offset, List<int> semanticsOffsets, List<dynamic> semanticsElements) {
-    if (recognizer != null && (recognizer is TapGestureRecognizer || recognizer is LongPressGestureRecognizer)) {
+    if (recognizer != null &&
+      (recognizer is TapGestureRecognizer || recognizer is LongPressGestureRecognizer)) {
```
Comment (chunhtai) [inline packages/flutter/lib/src/painting/text_span.dart]: same here

File: packages/flutter/lib/src/painting/text_span.dart [hunk 3]
```diff
@@ -414,7 +438,12 @@ class TextSpan extends InlineSpan {
   void debugFillProperties(DiagnosticPropertiesBuilder properties) {
     super.debugFillProperties(properties);
 
-    properties.add(StringProperty('text', text, showName: false, defaultValue: null));
+    properties.add(StringProperty(
+      'text',
+      text,
```
Comment (chunhtai) [inline packages/flutter/lib/src/painting/text_span.dart]: same here

File: packages/flutter/lib/src/rendering/paragraph.dart [hunk 4]
```diff
@@ -318,11 +320,12 @@ class RenderParagraph extends RenderBox
     assert(constraints != null);
     assert(constraints.debugAssertIsValid());
     _layoutTextWithConstraints(constraints);
-    // TODO(garyq): Since our metric for ideographic baseline is currently inacurrate
-    // and the non-alphabetic baselines are based off of the alphabetic baseline, we
-    // use the alphabetic for now to produce correct layouts. We should eventually change
-    // this back to pass the `baseline` property when the ideographic baseline is properly
-    // implemented (https://github.com/flutter/flutter/issues/22625).
+    // TODO(garyq): Since our metric for ideographic baseline is currently
```
Comment (chunhtai) [inline packages/flutter/lib/src/rendering/paragraph.dart]: same here

File: packages/flutter/lib/src/rendering/paragraph.dart [hunk 5]
```diff
@@ -412,8 +415,16 @@ class RenderParagraph extends RenderBox
...
     while (child != null) {
       final TextParentData textParentData = child.parentData;
-      final Matrix4 transform = Matrix4.translationValues(textParentData.offset.dx, textParentData.offset.dy, 0.0)
-        ..scale(textParentData.scale, textParentData.scale, textParentData.scale);
+      final Matrix4 transform = Matrix4.translationValues(
+        textParentData.offset.dx,
+        textParentData.offset.dy,
+        0.0,
+      )
+        ..scale(
```
Comment (chunhtai) [inline packages/flutter/lib/src/rendering/paragraph.dart]: I am not sure if how we indent in this situation, do we indent in this case?

File: packages/flutter/test/widgets/text_test.dart [hunk 6]
```diff
@@ -616,7 +732,8 @@ void main() {
               child: Container(
                 // Each word takes up more than a half of a line. Together they
                 // wrap onto two lines, but leave a lot of extra space.
-                child: Text('twowordsthateachtakeupmorethanhalfof alineoftextsothattheywrapwithlotsofextraspace',
+                child: Text('twowordsthateachtakeupmorethanhalfof alineoftextsot'
```
Comment (chunhtai) [inline packages/flutter/test/widgets/text_test.dart]: the style is a bit off

File: packages/flutter/lib/src/painting/text_span.dart (hunk 1 above)
Comment (Piinks) [reply packages/flutter/lib/src/painting/text_span.dart]: Fixed.

File: packages/flutter/lib/src/painting/text_span.dart (hunk 2 above)
Comment (Piinks) [reply packages/flutter/lib/src/painting/text_span.dart]: Where? I do not see it. :)

File: packages/flutter/lib/src/painting/text_span.dart (hunk 3 above)
Comment (Piinks) [reply packages/flutter/lib/src/painting/text_span.dart]: I'm missing it here as well. Where? :)

File: packages/flutter/lib/src/rendering/paragraph.dart (hunk 4 above)
Comment (Piinks) [reply packages/flutter/lib/src/rendering/paragraph.dart]: Fixed.

File: packages/flutter/lib/src/rendering/paragraph.dart (hunk 5 above)
Comment (Piinks) [reply packages/flutter/lib/src/rendering/paragraph.dart]: Fixed.

File: packages/flutter/test/widgets/text_test.dart (hunk 6 above)
Comment (Piinks) [reply packages/flutter/test/widgets/text_test.dart]: Fixed.

File: packages/flutter/lib/src/painting/text_span.dart (hunk 2 above)
Comment (chunhtai) [reply packages/flutter/lib/src/painting/text_span.dart]: hmmm I might be wrong but I remember it should be something like
```dart
if (
    recognizer != null &&
    (recognizer is TapGestureRecognizer || recognizer is LongPressGestureRecognizer)
)
```

File: packages/flutter/lib/src/painting/text_span.dart (hunk 3 above)
Comment (chunhtai) [reply packages/flutter/lib/src/painting/text_span.dart]: I might be wrong, but do we need StringProperty in a new line?
```dart
properties.add(
    StringProperty(
        'text',
         ....
     )
);
```

**Comment by Piinks** [comment]
Comments addressed. Thanks @chunhtai!

File: packages/flutter/lib/src/rendering/paragraph.dart [hunk 7]
```diff
@@ -887,18 +907,37 @@ class RenderParagraph extends RenderBox
...
+    ];
   }
 
   @override
   void debugFillProperties(DiagnosticPropertiesBuilder properties) {
     super.debugFillProperties(properties);
     properties.add(EnumProperty<TextAlign>('textAlign', textAlign));
     properties.add(EnumProperty<TextDirection>('textDirection', textDirection));
-    properties.add(FlagProperty('softWrap', value: softWrap, ifTrue: 'wrapping at box width', ifFalse: 'no wrapping except at line break characters', showName: true));
+    properties.add(FlagProperty(
```
Comment (chunhtai) [inline packages/flutter/lib/src/rendering/paragraph.dart]: here as well

File: packages/flutter/lib/src/rendering/paragraph.dart [hunk 8]
```diff
@@ -887,18 +907,37 @@ class RenderParagraph extends RenderBox
...
+      'softWrap',
+      value: softWrap,
+      ifTrue: 'wrapping at box width',
+      ifFalse: 'no wrapping except at line break characters',
+      showName: true,
+    ));
     properties.add(EnumProperty<TextOverflow>('overflow', overflow));
-    properties.add(DoubleProperty('textScaleFactor', textScaleFactor, defaultValue: 1.0));
-    properties.add(DiagnosticsProperty<Locale>('locale', locale, defaultValue: null));
+    properties.add(DoubleProperty(
```
Comment (chunhtai) [inline packages/flutter/lib/src/rendering/paragraph.dart]: same

File: packages/flutter/lib/src/rendering/paragraph.dart [hunk 9]
```diff
@@ -887,18 +907,37 @@ class RenderParagraph extends RenderBox
...
+      'textScaleFactor',
+      textScaleFactor,
+      defaultValue: 1.0,
+    ));
+    properties.add(DiagnosticsProperty<Locale>(
```
Comment (chunhtai) [inline packages/flutter/lib/src/rendering/paragraph.dart]: same

**Review by chunhtai — APPROVED** [review]
LGTM

**Pull request merged**